### PR TITLE
Fix broken managed identity links

### PR DESCRIPTION
**Pull request by jkatz35**
Reroutes links to how to guides for managed identity resource assignment to the new correct location

### CODE REVIEW

**Comment by Jak-MS** [comment]
@barclayn 
Can you review the proposed changes? 

Note: the links will need to be fixed before we can merge it. 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/identity/managed-identities-azure-resources/index.yml [hunk 1]
```diff
@@ -59,8 +59,8 @@ landingContent:
       - linkListType: how-to-guide
         links:
           - text: Portal
-            url: howto-assign-access-portal.md
+            url: how-to-configure-managed-identities-scale-sets?pivots=identity-mi-methods-azp
```
Comment (cilwerner) [inline docs/identity/managed-identities-azure-resources/index.yml:62]:
```suggestion
            url: how-to-configure-managed-identities-scale-sets?pivots=identity-mi-methods-azp.md
```

File: docs/identity/managed-identities-azure-resources/index.yml [hunk 2]
```diff
@@ -59,8 +59,8 @@ landingContent:
...
           - text: CLI
-            url: howto-assign-access-cli.md
+            url: how-to-assign-access-azure-resource?pivots=identity-mi-access-cli
```
Comment (cilwerner) [inline docs/identity/managed-identities-azure-resources/index.yml:64]:
```suggestion
            url: how-to-assign-access-azure-resource?pivots=identity-mi-access-cli.md
```

File: docs/identity/managed-identities-azure-resources/index.yml [hunk 3]
```diff
@@ -59,8 +59,8 @@ landingContent:
...
           - text: PowerShell
-            url: howto-assign-access-powershell.md
+            url: hhow-to-assign-access-azure-resource?pivots=identity-mi-access-powershell
```
Comment (cilwerner) [inline docs/identity/managed-identities-azure-resources/index.yml:66]:
```suggestion
            url: hhow-to-assign-access-azure-resource?pivots=identity-mi-access-powershell.md
```

**Review by cilwerner — CHANGES_REQUESTED** [review]
.md file extensions needed

**Review by barclayn — APPROVED** [review]

**Comment by barclayn** [comment]
#sign-off

File: docs/identity/managed-identities-azure-resources/index.yml [hunk 4]
```diff
@@ -59,8 +59,8 @@ landingContent:
       - linkListType: how-to-guide
         links:
           - text: Portal
-            url: howto-assign-access-portal.md
+            url: how-to-configure-managed-identities-scale-sets?pivots=identity-mi-methods-azp.md
```
Comment (Jak-MS) [inline docs/identity/managed-identities-azure-resources/index.yml:62]:
```suggestion
            url: how-to-configure-managed-identities-scale-sets.md?pivots=identity-mi-methods-azp
```

File: docs/identity/managed-identities-azure-resources/index.yml [hunk 5]
```diff
@@ -59,8 +59,8 @@ landingContent:
...
           - text: CLI
-            url: howto-assign-access-cli.md
+            url: how-to-assign-access-azure-resource?pivots=identity-mi-access-cli.md
```
Comment (Jak-MS) [inline docs/identity/managed-identities-azure-resources/index.yml:64]:
```suggestion
            url: how-to-assign-access-azure-resource.md?pivots=identity-mi-access-cli
```

File: docs/identity/managed-identities-azure-resources/index.yml [hunk 6]
```diff
@@ -59,8 +59,8 @@ landingContent:
...
           - text: PowerShell
-            url: howto-assign-access-powershell.md
+            url: hhow-to-assign-access-azure-resource?pivots=identity-mi-access-powershell.md
```
Comment (Jak-MS) [inline docs/identity/managed-identities-azure-resources/index.yml:66]:
```suggestion
            url: hhow-to-assign-access-azure-resource.md?pivots=identity-mi-access-powershell
```

File: docs/identity/managed-identities-azure-resources/index.yml [hunk 7]
```diff
@@ -59,8 +59,8 @@ landingContent:
...
         links:
           - text: Portal
-            url: howto-assign-access-portal.md
+            url: how-to-configure-managed-identities-scale-sets.md?pivots=identity-mi-methods-azp
           - text: CLI
-            url: howto-assign-access-cli.md
+            url: how-to-assign-access-azure-resource.md?pivots=identity-mi-access-cli
           - text: PowerShell
-            url: howto-assign-access-powershell.md
+            url: hhow-to-assign-access-azure-resource.md?pivots=identity-mi-access-powershell
```
Comment (Jak-MS) [inline docs/identity/managed-identities-azure-resources/index.yml:66]:
```suggestion
            url: how-to-assign-access-azure-resource.md?pivots=identity-mi-access-powershell
```

**Pull request merged**